connector/servicegraphconnector/connector_test.go

```diff
@@ -606,6 +606,7 @@ func TestExtraDimensionsLabels(t *testing.T) {
 		Dimensions:              extraDimensions,
 		LatencyHistogramBuckets: []time.Duration{time.Duration(0.1 * float64(time.Second)), time.Duration(1 * float64(time.Second)), time.Duration(10 * float64(time.Second))},
 		Store:                   StoreConfig{MaxItems: 10},
+		MetricsFlushInterval:    &[]time.Duration{0}[0],
```

Comment:
I don't see any other usages of this pattern throughout the codebase, but various pointer functions in the style below are present. This is not particularly readable, so IMO it should be changed.

```
func ptr[T any](value T) *T {
	return &value
}
```

Reply:
Fair point, changed.